internal/pipeline/resolver/repository.go

```diff
@@ -82,12 +81,16 @@ func filterPipelines(repoURLs []string, org string, client *buildkite.Client) ([
 
 func getRepoURLs(r *git.Repository) ([]string, error) {
 	if r == nil {
-		return nil, fmt.Errorf("could not determine current repository")
+		return nil, nil // could not resolve to any repository, proceed to another resolver
 	}
```

Comment:
@lizrabuya will this work the way we expect by checking here? It looks like it would still end up making some API calls that are redundant if we don't know the repository.

Line https://github.com/buildkite/cli/pull/266/files#diff-5fe66e65fdcc424120bfcc3ffe59abe20947c796250173ec0b13378dc4f28845R42 only checks if this returns an error, otherwise it proceeds.

I think we should check for a repository on line 20 or so probably